src/Helper.php

```diff
@@ -12,6 +12,7 @@ class Helper
 {
     const TRACK_HASH = "00032020";
     const ALBUM_HASH = "0004206c";
+    const PLAYLIST_HASH = "0006006c";
```

Comment:
Is there a "known source" for these hashes @duncan3dc? It seems like a whole host of different ones work for playlists;

```
0004006c
0006006c
10050024
10060a6c
```

The playlist hash I used here is from [this issue](https://github.com/SoCo/SoCo/issues/247#issue-46187362) I found, however the Sonos desktop application seems to use `10060a6c` with my Sonos 🤔.

Reply:
Know they're complete black magic. For Sonos resources (local media library) they're pretty reliable, but as soon as you move to streaming services there's a range in use with no obvious pattern (as you've discovered).

I've made several unsuccessful attempts to figure this out in the past to support Spotify albums. Last time I checked none of the libraries for other languages handled this successfully/reliably, but you might wanna check out their recent versions to see if they've cracked it.

Alternatively I've started a new library ([sonos-cloud](https://github.com/duncan3dc/sonos-cloud)) which uses the recently published public API for Sonos. This is obviously a lot more reliable than reverse engineering their SOAP calls, but I've not built much with it yet to know if it supports your use case here